agent/src/containeroptions/containeroptionparser.cpp

```diff
@@ -44,6 +44,31 @@ void ContainerOptionParser::readConfigElement(const json_t *element)
     }
 
     m_options->setEnableWriteBuffer(wo);
+
+    if (wo == true)
+    {
+        bool enableTemporaryFilesystemWriteBuffer = false;
+        if(!JSONParser::read(element,
+                             "enableTemporaryFileSystemWriteBuffer",
+                             enableTemporaryFilesystemWriteBuffer)) {
+            std::string errorMessage("Could not parse config due to: 'enableTemporaryFilesystemWriteBuffer' not found.");
+            log_warn() << errorMessage;
+        }
+        m_options->setEnableTemporaryFileSystemWriteBuffers(enableTemporaryFilesystemWriteBuffer);
+
+        if (enableTemporaryFilesystemWriteBuffer) {
+            int temporaryFileSystemSize = 100 * 1024 * 1024;
```

Comment:
It can't be a constexpr as it's set to a value on the line below this comment. The value here is just a "sane default" in case JSONParser fails for some reason.